sentinel/tests/unit/schedule/outbound.js

```diff
@@ -0,0 +1,522 @@
+const assert = require('chai').assert;
+const sinon = require('sinon');
+
+const config = require('../../../src/config'),
+      db = require('../../../src/db');
+
+const rewire = require('rewire');
+const outbound = rewire('../../../src/schedule/outbound');
+
+describe('outbound', () => {
```

Comment:
Since there are no other tests for schedules, would you mind being explicit that these are for `outbound schedule`? I find the two outbound objects easy to confuse.

Reply:
no problem doing that, but fwiw other people would almost certainly tell me to remove the 'schedule' because it's obvious from the directory 🤷‍♂

Reply:
you don't see the directory in the log output for the test runs...